Sources/Turf/CoreLocation.swift

```diff
@@ -50,6 +50,20 @@ extension CLLocationDirection {
     }
 }
 
+extension CLLocationCoordinate2D: Codable {
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.unkeyedContainer()
+        try container.encode(longitude)
+        try container.encode(latitude)
```

Comment:
The GeoJSON specification [allows](https://tools.ietf.org/html/rfc7946#section-3.1.1) for a third item in the array, representing the altitude or elevation of the position. As far as I know, Turf.js doesn’t make use of altitude values, but [Mapbox Java Services does support deserializing them](https://github.com/mapbox/mapbox-java/blob/c15c234837c1379d4c7044e9b4c769e0f3d10b39/services-geojson/src/main/java/com/mapbox/geojson/Point.java#L154).

I think this code will gracefully ignore any altitude present in the original GeoJSON, but it won’t round-trip the value. I think that’s fine for now, but FYI an alternative would be to decode the position as a CLLocation, which can store an altitude.

Reply:
When you create a CLLocation with `CLLocation(latitude:longitude:)`, `timestamp` is set to the current time. I suppose that would be problematic.

We could roll our own type instead of piggybacking on Core Location, but CLLocationCoordinate2D and CLLocation come out of the box with nice debugging tools and developer familiarity.

Let’s stick with CLLocationCoordinate2D until we implement a function for which the altitude becomes relevant. I think it’s fine if the GeoJSON serialization comes with a few caveats, since this library isn’t a general-purpose GeoJSON converter.